MyOwnORM/MyOwnORM/Service/CustomDbSetService.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbSetService<T> where T : class
```

Comment:
As I see this class works almostly with queries in string format so give it appropriate name

Reply:
He works also with expressions and other logic